shared/src/main/scala/akkaviz/protocol/package.scala

```diff
@@ -69,21 +71,25 @@ package object protocol {
 
   object IO {
 
-    def readServer(json: String): ApiServerMessage = {
-      upickle.default.read[ApiServerMessage](json)
-    }
+    import boopickle.Default._
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/lustefaniak/akka-viz/file/2145592517/issues/source?bid=3121282&fileBranchId=3151479#l74)